products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx

```diff
@@ -0,0 +1,246 @@
+---
+id: intro
+title: Securing vWAN
+sidebar_label: Securing vWAN
+hide_title: true
+description: Securing vWAN
+keywords:
+  - pan-os
+  - panos
+  - azure
+  - api
+  - firewall
+  - configuration
+  - terraform
+  - cloudngfw
+---
+
+
+# Deploying Cloud NGFW for Azure using Terraform
+
+
+## Objective
+
+[Cloud NGFW for Azure](https://www.paloaltonetworks.com/network-security/cloud-ngfw-for-azure) by Palo Alto Networks is natively integrated into the Azure Resource Manager framework and does not require the use of any vendor-specific APIs for deployment. 
+
+In this blog, you will learn how to deploy and configure the Cloud NGFW resource using Terraform. Since Cloud NGFW is directly exposed in the Azure Portal and Azure APIs as a native service, it only requires the [Azure Terraform Provider](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs) to deploy and configure the resource.
+
+For this deployment, we will create 2x Cloud NGFW resources to secure an Azure vWAN. One Cloud NGFW will be managed by Azure Rulestack, while the other will be managed by Panorama.
+
+The following resources will be created:
+- Resource group
+- Public IP address x2
+- Virtual WAN
+- Local Rulestack
+- Cloud NGFW x2
+- Routing Intent x2
+
+First, in a separate directory create base Terraform files: main.tf, variables.tf and terraform.tfvars.
+
+
+
+```hcl title="terraform.tfvars"
+region1 = "eastus"
+region2 = "westeurope"
+rg = "terraform-rg"
+panorama-string = "<Panorama registration string"
+```
+
+
+```hcl title="variables.tf"
+variable "rg" {
+  type = string
+}
+
+variable "region1" {
+  type = string
+}
+
+variable "region2" {
+  type = string
+}
+
+variable "panorama-string" {
+  type = string
+}
+```
+
+The features block is mandatory to run any azurerm deployment. If you’d like to stick with the defaults, leave it empty
+
+
+```hcl title="main.tf"
+provider "azurerm" {
+    features {}
+}
+```
+1. Resource Group.
+
+Query an existing Resource Group:
+```
+data "azurerm_resource_group" "rg" {
+  name = var.rg
+}
+```
+Or create a new one: 
+
+```
+resource "azurerm_resource_group" "rg" {
+  name     = "terraform-rg"
+  location = "East US"
+}
+```
+2. Public IP address
+
+Use the Public IP address resource to create at least one IP address that will be associated with the Cloud NGFW
```

Comment:
In the objectives section we mentioned we are going to create 2 x PIPs. Perhaps remove "at least one IP address"?

Reply:
fixed